state/model_test.go

```diff
@@ -560,20 +560,19 @@ func (s *ModelSuite) TestAllEndpointBindings(c *gc.C) {
 	c.Assert(err, jc.ErrorIsNil)
 	c.Assert(listBindings, gc.HasLen, 1)
 
-	c.Assert(listBindings[0], jc.DeepEquals, state.ApplicationEndpointBindings{
-		AppName: "wordpress",
-		Bindings: map[string]string{
-			"":                network.DefaultSpaceId,
-			"cache":           network.DefaultSpaceId,
-			"foo-bar":         network.DefaultSpaceId,
-			"db-client":       network.DefaultSpaceId,
-			"admin-api":       network.DefaultSpaceId,
-			"url":             network.DefaultSpaceId,
-			"logging-dir":     network.DefaultSpaceId,
-			"monitoring-port": network.DefaultSpaceId,
-			"db":              oneSpace.Id(),
-		},
-	})
+	expected := map[string]string{
+		"":                network.DefaultSpaceId,
+		"cache":           network.DefaultSpaceId,
+		"foo-bar":         network.DefaultSpaceId,
+		"db-client":       network.DefaultSpaceId,
+		"admin-api":       network.DefaultSpaceId,
+		"url":             network.DefaultSpaceId,
+		"logging-dir":     network.DefaultSpaceId,
+		"monitoring-port": network.DefaultSpaceId,
+		"db":              oneSpace.Id(),
+	}
+	c.Assert(listBindings[0].AppName, gc.Equals, app.Name())
+	c.Assert(listBindings[0].Bindings.Map(), gc.DeepEquals, expected)
```

Comment:
Should we have a `MapWithSpaceNames` test or do we have one already here?

Reply:
have one for the Bindings object: bindingsMockSuite.TestMapWithSpaceNames()